wdn/templates_4.1/includes/scriptsandstyles_speedy.shtml

```diff
@@ -0,0 +1,42 @@
+<noscript>
+    <link rel="stylesheet" href="https://cloud.typography.com/7717652/616662/css/fonts.css" />
+    <link rel="stylesheet" href="https://unlcms.unl.edu/wdn/templates_4.1/css/all.css?dep=$DEP_VERSION$" />
+</noscript>
+<style id="wdn-critical-styles">
+    <!--#include virtual="/wdn/templates_4.1/css/critical.css" -->
+</style>
+<script>
+    var WDNRemoveCriticalStyles = function() {
```

Comment:
I'd wrap all of this in an self-executing anonymous function to avoid polluting the global object with single use functions:
```
(function() {
...
})();


```